OpenRA.Utility/Program.cs

```diff
@@ -43,9 +43,16 @@ static void Main(string[] args)
 			Log.AddChannel("perf", null);
 			Log.AddChannel("debug", null);
 
-			Game.InitializeSettings(Arguments.Empty);
+			var arguments = new Arguments(args);
+			args = arguments.GetSkippedValues();
+
+			Game.InitializeSettings(arguments);
```

Comment:
What does this section have to do with settings?
This is mapping cli args to Arguments.